test/standalone/test-grpc-context.js

```diff
@@ -15,9 +15,10 @@
  */
 'use strict';
 
-var agent = require('../..').start({ samplingRate: 0 }).private_();
-
 var common = require('../hooks/common.js');
+
+var agent = require('../..')().startAgent({ samplingRate: 0 }).private_();
+
 var assert = require('assert');
 var express = require('../hooks/fixtures/express4');
```

Comment:
Does this loading of express/starting of the agent have to move to the before of the test below?